src/services/edusharing/index.js

```diff
@@ -35,21 +48,27 @@ class MerlinToken {
 
 module.exports = (app) => {
 	const eduSharingRoute = '/edu-sharing';
-	const merlinRoute = `${eduSharingRoute}/merlinToken`;
-	const docRoute = `${eduSharingRoute}/api`;
+	const eduSharingPlayerRoute = '/edu-sharing/player';
+	const merlinRoute = '/edu-sharing/merlinToken';
+	const docRoute = '/edu-sharing/api';
 
-	app.use(docRoute, staticContent(path.join(__dirname, '/docs/openapi.yaml')));
+	app.use(eduSharingRoute, new EduSharing(), (req, res) => {
```

Comment:
The service (nearly api) test are missing.

Reply:
@CeEv Which tests are needed? Where can I find existing tests? Is it really necessary to test this service?

Reply:
the tests are under test/services/edusharing.
You need to cover all new code.

Reply:
All new code is now covered